DuckDuckGo/AppDelegate.swift

```diff
@@ -45,6 +45,7 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
     
     func applicationDidBecomeActive(_ application: UIApplication) {
         startMigration()
+        TrackerLoader.shared.updateTrackers()
```

Comment:
> Do we need to be more explicit about when the tracker data is downloaded to prevent too much data usage or, conversely, getting out of date by using a cached version?
We're using a standard etag model for caching so it _should_ be ok. Have you noticed data being re-downloaded?